build_runner/lib/src/asset_graph/optional_output_tracker.dart

```diff
@@ -23,36 +23,44 @@ import 'node.dart';
 class OptionalOutputTracker {
   final _checkedOutputs = <AssetId, bool>{};
   final AssetGraph _assetGraph;
+  final List<String> _buildDirs;
   final List<BuildPhase> _buildPhases;
+  final String _rootPackage;
 
-  OptionalOutputTracker(this._assetGraph, this._buildPhases);
+  OptionalOutputTracker(
```

Comment:
Maybe expand or rephrase the doc comment on line 18?

Reply:
done